frontend/src/pages/spotnet/information/Informaion.jsx

```diff
@@ -1,41 +1,71 @@
-import './information.css'
+import './information.css';
 import { ReactComponent as Star } from "../../../assets/particles/star.svg";
-import React from "react";
+import React, { useEffect, useState } from "react";
 
-const Informaion = () => {
+const Information = () => {
+    const [data, setData] = useState({ total_opened_amount: 0, unique_users: 0 });
+    const [loading, setLoading] = useState(true);
+    const [error, setError] = useState(null);
+    
+    useEffect(() => {
+        const fetchData = async () => {
+            try {
+                const response = await fetch(`${process.env.REACT_APP_BACKEND_URL}/api/get_stats`);
+                if (!response.ok) {
+                    throw new Error('Failed to fetch data');
+                }
+                const result = await response.json();
+                setData({
+                    total_opened_amount: result.total_opened_amount,
+                    unique_users: result.unique_users,
+                });
+            } catch (error) {
+                setError(error.message);
+            } finally {
+                setLoading(false);
+            }
+        };
+    
+        fetchData();
+    }, []);
+    
     const starData = [
         { top: 9, left: -6.2, size: 20 },
-        { top: 9, left: 39, size: 15,},
+        { top: 9, left: 39, size: 15 },
         { top: -8, left: 85, size: 18 },
         { top: 74, left: 43, size: 22 },
-    ]
-    return(
+    ];
+
+    return (
         <div className="information">
             <div className="card-info__container">
                 <div className="card-info flex">
                     <h1>TVL</h1>
-                    <h3>$245.7k</h3>
+                    <h3>{loading ? "Loading..." : error ? `Error: ${error}` : data.total_opened_amount}</h3>
```

Comment:
Value should be formatted into such style `$245.7k`. Currently it seems to just display raw number from a backend